### PR TITLE
Increase language search bar width

### PR DESCRIPTION
**Pull request by 2580ayush2580**
It Increases the width of the language search.

before Pull Request:-

s5

After Pull Request:-

s3

Signed-off-by: Ayush Kumar [2580ayush2580@gmail.com]()

### CODE REVIEW

**Comment by SimonBrandner** [comment]
@2580ayush2580 I am not sure if this is desirable. One of the languages is for example `Norsk Nynorsk` the user may just search for `Nynorsk` and expect to find it.

**Comment by SimonBrandner** [comment]
While you are at it, you can increase the width of the search if you want. Some languages are cut off...

Screenshot_20210117_112851

**Comment by 2580ayush2580** [comment]
> @2580ayush2580 I am not sure if this is desirable. One of the languages is for example `Norsk Nynorsk`. The user may just search for `Nynorsk` and expect to find it.

okay sure, I will increase the width of the search bar.

**Comment by 2580ayush2580** [comment]
done!

s3

**Review by SimonBrandner — CHANGES_REQUESTED** [review]
@2580ayush2580 Could you please update the description?

Otherwise LGTM :+1:

File: res/css/views/auth/_LanguageSelector.scss [hunk 1]
```diff
@@ -23,6 +23,7 @@ limitations under the License.
     font-size: $font-14px;
     font-weight: 600;
     color: $authpage-lang-color;
+    width: 160px;
```
[the remark this answers is not in the document]
Comment (2580ayush2580) [reply res/css/views/auth/_LanguageSelector.scss:26]: If I remove this line then login page language search bar will have no width change.

s6

[the remark this answers is not in the document]
Comment (SimonBrandner) [reply res/css/views/auth/_LanguageSelector.scss:26]: Ok, sorry...

**Review by SimonBrandner — APPROVED** [review]

File: res/css/views/auth/_LanguageSelector.scss (hunk 1 above)
Comment (jryans) [inline res/css/views/auth/_LanguageSelector.scss:26]: Hmm, I don't think is quite what we want, as it means the menu is no longer centered in the column. If you can fix that up as well, then the larger widget could be okay.

**Review by jryans — CHANGES_REQUESTED** [review]

**Review by jryans — APPROVED** [review]
Looks good to me, thanks for working on it! 😄

**Pull request merged**